test/specs/apexLsp.e2e.ts

```diff
@@ -5,29 +5,106 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import { step } from 'mocha-steps';
+import { TextEditor } from 'wdio-vscode-service';
 import { TestSetup } from '../testSetup';
+import * as utilities from '../utilities';
 
 describe('Apex LSP', async () => {
   let testSetup: TestSetup;
   let projectName: string;
 
   step('Set up the testing environment', async () => {
-    testSetup = new TestSetup('ApexLsp', false);
+    utilities.log(`ApexLsp - Set up the testing environment`);
+    testSetup = new TestSetup('ApexLsp', true);
     await testSetup.setUp();
     projectName = testSetup.tempProjectName.toUpperCase();
+
+    // Create Apex Class
+    await utilities.createApexClassWithTest('ExampleClass');
+  });
+
+  step('Verify Extension is Running', async () => {
+    utilities.log(
+      `${testSetup.testSuiteSuffixName} - Verify Extension is Running`
+    );
+    // Using the Command palette, run Developer: Show Running Extensions
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'Developer: Show Running Extensions',
+      1
+    );
+
+    // Verify Apex extension is present and running
+    const extensionNameDivs = await $$('div.name');
+    let extensionWasFound = false;
+    for (const extensionNameDiv of extensionNameDivs) {
+      const text = await extensionNameDiv.getText();
+      if (text.includes('salesforce.salesforcedx-vscode-apex')) {
+        extensionWasFound = true;
+      }
+    }
+    expect(extensionWasFound).toBe(true);
   });
 
-  step('what?', async () => {
-    // TODO: implement
-    expect(1).toBe(1);
+  step('Go to Definition', async () => {
+    utilities.log(`${testSetup.testSuiteSuffixName} - Go to Definition`);
+    // Get open text editor
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+    const textEditor = (await editorView.openEditor(
+      'ExampleClassTest.cls'
+    )) as TextEditor;
```

Comment:
Hey Cristi, this looks good. More of a minor suggestion and less of a requirement would be to place this setup piece in a util or a pre hook to avoid repetition. I suspect this will be used quite frequently.

Reply:
I think we do use it frequently but also sometimes for different purposes, and wdio is just weird so I'd keep it in how it is in every step needed at least for now